### ISSUE TITLE
Accessibility cannot be loaded

### ISSUE THREAD

**Issue by ghost** [wmchris/DellXPS15-9550-OSX#185]
Has anyone experience an issue where opening the System Preferences -> Accessibility now loading, and then outputting an error?
macOS Monterrey 12.3.1

EDITED:
Actually it happens with any System Preference option. Sometimes. Not all of the time.
